admin-manual/installation-setup/customization/scaling-archivematica.rst

```diff
@@ -0,0 +1,396 @@
+
+The larger a Transfer or AIP is, the longer it will take to create the Index. 
+Some users have found that Indexing can fail on very large Transfers or AIPS 
+(e.g. with many thousands of files). 
+
+Use of Elasticsearch is optional. Installing Archivematica without 
+Elasticsearch means reduced consumption of compute resources and lower 
+operational complexity. Disabling Elasticsearch means that the Backlog, 
+Appraisal, and Archival Storage tabs do not appear and their functionality is 
+not available.
+
+See :ref:`Upgrade in indexless mode <upgrade-indexless>` for more details. 
+
+Allow indexing to fail
+++++++++++++++++++++++
+
+Indexing very large datasets can be so resource-intensive that indexing will 
+fail. By default, Archivematica will abort processing and invoked the 
+“Failed SIP” microservice. 
+
+The ``ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_INDEX_AIP_CONTINUE_ON_ERROR`` parameter 
+can be set to allow indexing to fail. When this is set and indexing fails, 
+the AIP will carry on with processing and be stored. It can’t be found using 
+the normal search features in the Appraisal and Archival Storage tabs. 
+
+This feature doesn’t optimize performance so much as mitigate performance 
+limitations. See `MCPClient`_ Configuration for details. 
+
+Disabling task output
++++++++++++++++++++++
+
+Archivematica allows users to configure their MCPClient(s) in order to control 
+whether or not output streams (stdout and stderr) from the client scripts are 
+captured and then passed from the task workers to the task manager 
+(the MCPServer). Disabling task output will provide performance improvements. 
+
+See :ref:`Task output capturing configuration <task-output-capturing-admin>`.
+
+Processing configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`Processing configuration <dashboard-processing>` screen provides many 
+options for controlling what actions Archivematica performs. The following 
+settings can improve performance. 
+
+**Select file format identification command (Transfer):** Using Siegfried for 
+file identification has been shown to be faster than Fido in this 
+`benchmarking`_ study.
+
+**Select file format identification command (Ingest):** If you are using the 
+Archivematica backlog and have accumulated items in the backlog for a long 
+period of time, e.g. months or even years, then you might want Archivematica to
+refresh its file format identification at the Ingest stage of the workflow. If
+Ingest is happening shortly after Transfer, selecting `Use existing data` 
+should be adequate and will save processing time. 
+
+**Generate thumbnails:** If you are generating thumbnails, selecting `yes 
+without defaults` will only generate thumbnail images for formats that have a 
+specific thumbnail rule defined. The default thumbnail rule only produces a 
+grey icon and for many formats has little value, but in transfers with many 
+files, can take significant processing time. 
```

Comment:
Worth adding here that you can also disable generation of thumbnails all together?

Reply:
yes, have added now.